File: Python/7py 2/words_to_sentence.py
"""
|--------------------------------------------------------------------------
| Words to sentence
|--------------------------------------------------------------------------
|
"""

def words_to_sentence(words):
   return ' '.join(words)


def words_to_sentence(words):
   res = ""
   for word in words:
      res += word + ' '
   res = res[:-1]
   return res


words_to_sentence = ' '.join


def words_to_sentence(words):
   x = ''
   for i in words:
      x += i + ' '
   return x.strip(' ')


print(words_to_sentence(['bacon', 'is', 'delicious']), 'bacon is delicious')

File: JavaScript/7js 3/wordsToSentence.js
/*
|--------------------------------------------------------------------------
| Words to sentence
|--------------------------------------------------------------------------
|
*/

function wordsToSentence(words) {
   return words.join(' ');
}


const wordsToSentence = require('lodash/fp/join')(' ');


function wordsToSentence(words) {
   var x = "";
   for (let i = 0; i < words.length; i++) {
      x = x + words[i] + " ";
   }
   return x.trim();
}


function wordsToSentence(words) {
   return words.reduce(function (prev, curr) {
      return prev + ' ' + curr
   });
}


console.log(wordsToSentence(["hello", "world"]), "hello world");